lsp/gnls.lua

```diff
@@ -0,0 +1,5 @@
+return {
```

Comment:
missing doc. look at other configs.

Reply:
I'm still missing something. Let me have a look. sorry for the noise.